examples/pytorch/question-answering/run_qa.py

```diff
@@ -56,10 +57,18 @@ class ModelArguments:
     """
     Arguments pertaining to which model/config/tokenizer we are going to fine-tune from.
     """
-
     model_name_or_path: str = field(
         metadata={"help": "Path to pretrained model or model identifier from huggingface.co/models"}
     )
+    teacher_model_name_or_path: Optional[str] = field(
```

Comment:
nit can we change to distill_teacher and keep the documentation. I don't think we need name_or_path on there and makes it a very long variable to type out for running